airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -613,6 +613,17 @@ definitions:
           - "%Y-%m-%dT%H:%M:%S.%f%z"
           - "%Y-%m-%d"
           - "%s"
+      start_datetime:
```

Comment:
Just changed the order as fields above `start_datetime` where not mandatory anymore